hera_cal/delay_filter.py

```diff
@@ -91,7 +91,7 @@ def get_filled_data(self):
         return filled_data, filled_flags
 
     def write_filtered_data(self, res_outfilename=None, CLEAN_outfilename=None, filled_outfilename=None, filetype='uvh5',
-                            partial_write=False, clobber=False, add_to_history='', **kwargs):
+                            partial_write=False, clobber=False, add_to_history='', extra_attrs={}, **kwargs):
```

Comment:
I don't understand why we need kwargs for write_data. Which pyuvdata kwargs would you envision using that you don't think we should pass explicitly?

Reply:
One example is the newly added `chunks` parameter, but also the HDF5 compression parameters, etc. Things that the average user would use the defaults in pyuvdata, but for us we might want access to it. It also gives us automatic access to any future kwarg in pyuvdata that we don't have explicitly here